docs/source/algorithms/PolarizationCorrectionFredrikze-v1.rst

```diff
@@ -1,3 +1,7 @@
+To update your documentation based on the new approach of using spin states directly rather than flipper configurations, you need to revise the relevant sections to reflect this change. Here’s an updated version of your documentation with a focus on the new spin state configurations and removing references to flipper configurations:
+
+---
```

Comment:
We should highlight things like this in the release notes rather than at the top of the docs, which have a standardised style.
```suggestion
```